lib/dialects/sqlite/query-interface.js

```diff
@@ -39,7 +40,7 @@ class SQLiteQueryInterface extends QueryInterface {
     options = options || {};
 
     const fields = await this.describeTable(tableName, options);
-    fields[attributeName] = this.normalizeAttribute(dataTypeOrOptions);
+    _.assign(fields[attributeName], this.normalizeAttribute(dataTypeOrOptions));
```

Comment:
Why `assign` ?

Reply:
To not overwrite existing properties on the `fields[attributeName]` object. Effectively like applying a delta. `Object.assign()` would also work.

Reply:
Use `Object.assign`